scipy/spatial/transform/rotation.py

```diff
@@ -43,7 +45,8 @@ def __init__(self, quat, normalized=False):
             # which (0,0,0,1) in (x,y,z,w) format
             zero_norms = norms == 0
             if zero_norms.any():
-                warnings.warn("Found zero norm quaternions in input, replacing with identity quaternions.")
+                warnings.warn("""Found zero norm quaternions in input,
+                        replacing with identity quaternions.""")
                 self._quat[zero_norms] = np.array([0, 0, 0, 1])
```

Comment:
I would suggest to simply break a standard `"..."` string, at least this is how I usually saw it's done.

Also check https://www.python.org/dev/peps/pep-0008/#indentation for formatting guidelines.

Reply:
Breaking a standard `"..."` string with either `\` or using a multi-line string with triple quotes such as 
`'''abc`
`def'''` 
prints newlines and blank spaces also. This method prints the string on a single line.

Do correct me if there's some other method I'm missing.

Reply:
Well, in this case you should write it as:

```
warnings.warn("Found zero norm quaternions in input, 
               replacing with identity quaternions.")
```

Check that it works as expected.

Reply:
That doesn't seem to work:
`SyntaxError: EOL while scanning string literal`

Reply:
Oh, I'm sorry. What I actually meant:
```Python
warnings.warn("Found zero norm quaternions in input, "
              "replacing with identity quaternions.")
```

Reply:
Ah, that works. I'd only seen triple quoted string broken this way.